app/views/assistance_requests/index.html.erb

```diff
@@ -1,55 +1,63 @@
-<ul class="media-list requests-list"></ul>
+<div class="requests-list"></div>
 
 <script id="request_template" type="x-tmpl-mustache">
-  <li class="media">
-    <img src="{{requestor.avatar_url}}" class="media-object" />
-    <div class="media-body">
-      <h4 class="media-heading">{{requestor.first_name}} {{requestor.last_name}}</h4>
-      <p>Requested assistance <abbr class="timeago" title="{{start_at}}">{{start_at}}</abbr></p>
+  <li>
+    <div class="student-avatar">
+      <img src="{{requestor.custom_avatar.thumb.url}}{{^requestor.custom_avatar.thumb.url}}{{requestor.avatar_url}}{{/requestor.custom_avatar.thumb.url}}" />
```

Comment:
Won't the very first first {{}} in this line fail if they don't have a custom avatar thumb?